### PR TITLE
Improve docs for Std library [WIP]

### PR DESCRIPTION
**Pull request by sylwiabr**
### Pull Request Description

- Improve documentation for Std library

### Important Notes

- Mention important elements of the design.
- Mention any notable changes to APIs. 

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md).
- [x] The code has been tested where possible.

### CODE REVIEW

File: stdlib/Std/src/Geo.luna [hunk 1]
```diff
@@ -9,18 +9,68 @@ class GeoPoint:
     def shortRep: "(" + self.latitude.toText + ", " + self.longitude.toText + ")"
     def toFeature: GeoJSONFeature JSON.empty self.toJSON
 
+# GeoJSON is a format for encoding a variety of geographic data structures. 
+# GeoJSONFeatures contain a geometry and additional properties
+# GeoJSON supports the following geometry types: Point, LineString, Polygon, MultiPoint, MultiLineString, and MultiPolygon.
```
Comment (iamrecursion) [inline stdlib/Std/src/Geo.luna]: Can we wrap this to 80 characters?

File: stdlib/Std/src/Time.luna [hunk 2]
```diff
@@ -3,50 +3,165 @@ import Std.Base
...
     # Is this timezone valid only throughout the summertime?
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    justSummer = pacificStandardTime.summerOnly
+    # >    None
+    #
+    # `return`: Boolean value describing if the timezone has daylight saving time or not.
```
Comment (iamrecursion) [inline stdlib/Std/src/Time.luna]: This is unclear to me. Is it whether the provided TZ can be in daylight savings time, or whether it currently is? I can see from the code, but the wording is unclear.

File: stdlib/Std/src/Time.luna [hunk 3]
```diff
@@ -3,50 +3,165 @@ import Std.Base
...
     # The three letter shortcut name for this timezone, can be blank.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    timeZoneName = pacificStandardTime.name
+    # >    None
+    #
+    # `return`: Text value contains the three letter shortcut name for this timezone, can be blank.
```
Comment (iamrecursion) [inline stdlib/Std/src/Time.luna]:
```suggestion
    # `return`: Text value containing the three letter short name for this 
    #           timezone, can be blank.
```

File: stdlib/Std/src/Time.luna [hunk 4]
```diff
@@ -3,50 +3,165 @@ import Std.Base
...
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    newTimeZone = TimeZone.fromMinutes 360
+    # >    None
+    #
+    # `m`: The offset for new timezone in minutes.
+    #
+    # `return`: New nameless timezone instance with the offset `m` in minutes.
```
Comment (iamrecursion) [inline stdlib/Std/src/Time.luna]:
```suggestion
    # `return`: New unnamed timezone instance with the offset `m` in minutes.
```

File: stdlib/Std/src/Time.luna [hunk 5]
```diff
@@ -3,50 +3,165 @@ import Std.Base
...
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    newTimeZone = TimeZone.fromHours 4
+    # >    None
+    #
+    # `h`: The offset for new timezone in hours.
+    #
+    # `return`: New nameless timezone instance with the offset `h` in hours.
```
Comment (iamrecursion) [inline stdlib/Std/src/Time.luna]:
```suggestion
    # `return`: New unnamed timezone instance with the offset `h` in hours.
```

File: stdlib/Std/src/Time.luna [hunk 6]
```diff
@@ -3,50 +3,165 @@ import Std.Base
...
+    # Returns the `TimeZone` object representing the GMT timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    gmtTimeZone = TimeZone.gmt
+    # >    None
+    #
+    # `return`: The `TimeZone` object representing the GMT timezone.
     def gmt: TimeZoneVal 0 False "GMT"
```
Comment (iamrecursion) [inline stdlib/Std/src/Time.luna]: It seems really strange to me that we have constants for some time-zones and not others. `UTC` I can understand, but why do these others get special treatment?

Comment (sylwiabr) [reply stdlib/Std/src/Time.luna]: I have no idea, probably this was just POC. I can add more time-zones but I think it should go in separate PR.

Comment (iamrecursion) [reply stdlib/Std/src/Time.luna]: Oh definitely if we want to add more.

**Pull request merged**